admin/thirdparty/chosen/chosen/chosen.jquery.js

```diff
@@ -49,7 +49,8 @@
 
     SelectParser.prototype.add_option = function(option, group_position, group_disabled) {
       if (option.nodeName.toUpperCase() === "OPTION") {
-        if (option.text !== "") {
+        // workaround for https://github.com/harvesthq/chosen/issues/2125
+        if (!option.text.match(/^\s*$/g)) {
```

Comment:
We can't do this, we've had this problem too much with dropdown.

Some people do know coffeescript and fixed our coffee script and compiled, some people haven't and did this kind of fix and then coffee script got fixed again and compiled over the changes.

NO "FIXING" EXTERNAL LIBS PLEASE

May we don't fix this issue in this PR and instead we replace chosen with select2 in another PR for 4.0?